test/tests/LayerTest.coffee

```diff
@@ -1144,3 +1144,34 @@ describe "Layer", ->
 			a1.isAnimating.should.equal false
 			a2.isAnimating.should.equal false
 			a3.isAnimating.should.equal true
+
+
+	describe "Point conversion", ->
```

Comment:
Nested layers can be tested. Canvas conversions are hard to test since these depend on the window size which depends on the testing setup.